superset/utils/core.py

```diff
@@ -1025,8 +1025,8 @@ def get_filter_key(f: Dict[str, Any]) -> str:
         for existing in adhoc_filters:
             if (
                 existing["expressionType"] == "SIMPLE"
-                and existing["comparator"] is not None
-                and existing["subject"] is not None
+                and "comparator" in existing
+                and "subject" in existing
```

Comment:
Maybe use `existing.get("subject") is not None` to make sure the logic is still the same?

Reply:
OK, I made this change